content/docs/command-reference/plots/index.md

```diff
@@ -286,19 +286,19 @@ Download the Vega-Lite libraries into the directory where you'll produce the
 `dvc plots`:
 
 ```dvc
-$ wget https://cdn.jsdelivr.net/npm/vega@5.10.0 -O my_vega.js
-$ wget https://cdn.jsdelivr.net/npm/vega-lite@4.8.1 -O my_vega_lite.js
-$ wget https://cdn.jsdelivr.net/npm/vega-embed@6.5.1 -O my_vega_embed.js
+$ wget https://cdn.jsdelivr.net/npm/vega@5.20.2 -O my_vega.js
+$ wget https://cdn.jsdelivr.net/npm/vega-lite@5.1.0 -O my_vega_lite.js
+$ wget https://cdn.jsdelivr.net/npm/vega-embed@6.18.2 -O my_vega_embed.js
 ```
 
 Create the following HTML file and save it in `.dvc/plots/mypage.html`:
 
 ```html
 <html>
   <head>
-    <script src="my_vega.js" type="text/javascript"></script>
-    <script src="my_vega_lite.js" type="text/javascript"></script>
-    <script src="my_vega_embed.js" type="text/javascript"></script>
+    <script src="../my_vega.js" type="text/javascript"></script>
+    <script src="../my_vega_lite.js" type="text/javascript"></script>
+    <script src="../my_vega_embed.js" type="text/javascript"></script>
```

Comment:
after iterative/dvc#6431, we make a webpage in directory, it would probably be the best to put here some placeholder for user to fill, @jorgeorpinel do you think something like `/full/path/to/my_vega.js` would be good? Or should we use something more, "eye-bothering" like {FULL_PATH}?

Reply:
`../path/to/my_vega.js` should be clear enough here @pared. Thanks